src/ORM/Query.php

```diff
@@ -1275,4 +1275,22 @@ protected function _decorateResults($result)
 
         return $result;
     }
+
+    /**
+     * Selects all fields in the schema with the exception of the fields you pass
+     * in the $excludedFields parameter.
+     *
+     * @param array $excludedFields fields you don't want to select
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAllExcept(array $excludedFields = [], $overwrite = false)
```

Comment:
Yes, I understood that, but I think the fallback is not a great idea. If you call this method after calling join() or contain()!it would read as you are selecting all from all tables. But if we make the first param the table object there can be no confusion